setup/src/Magento/Setup/Test/Unit/Module/Setup/SetupCacheTest.php

```diff
@@ -60,7 +60,7 @@ public function testGetNonexistent($field)
     public function getNonexistentDataProvider()
     {
         return [
-            [null],
+            [''],
```

Comment:
What is the reason to change that ?

Reply:
@xmav Hello! Thanks for your review, I reverted this change.